### ISSUE TITLE
Screenshot cannot be saved in base64 format

### ISSUE THREAD

**Issue by ankugarg** [extent-framework/extentreports-java#59]
Using:
Extent reports - 4.0.9
Klov server - 0.2.5

Seems to be failing due to below code: Function path - com.aventstack.extentreports.mediastorage.impl.HttpMediaManagerImplKlov.storeMedia(Media) 
`if (m.getPath() == null || m.getBase64String() != null)
            return;`

In case of base 64 `m.getPath`  will always return null

**Comment by foursyth** [comment]
That is correct, because a base64 screenshot does not require an upload (since its part of the details).

Btw, are you attaching the screenshot to test or log?

**Comment by ankugarg** [comment]
@foursyth 

Trying to attach it to test.

Do you know how to build `extentreports-java` project? If yes, please share.

**Comment by anshooarora** [comment]
Is there any exception thrown?

**Comment by ankugarg** [comment]
@anshooarora 

No error is thrown.

**Comment by anshooarora** [comment]
I am able to repro, issue on Klov side.

**Comment by ankugarg** [comment]
@anshooarora 

Can you please tell when would you be able to fix this issue? I know this is community edition so you might have some other priority work.

Also, can you please share how to build the project?

**Comment by ankugarg** [comment]
@anshooarora 

Did you get a chance to work on it?

**Comment by virenv** [comment]
@ankugarg : Let me take a look at the issue over the weekend. I will get back to you.

**Comment by ankugarg** [comment]
@virenv  - were you able to look into the issue?

**Comment by PratibhaRamteke** [comment]
I am also facing the same issue. Anyone could help.

**Comment by anshooarora** [comment]
Note:  adding base64 strings to logs works today:

```
test.fail("base64", MediaEntityBuilder.createScreenCaptureFromBase64String(base64).build());
```

Adding to `test` will be included extentreports-4.10.0 onwards, I will be posting the snapshot version soon.
